src/templates/ru/2020/methodology.html

```diff
@@ -0,0 +1,478 @@
+          <source type="image/webp" srcset="/static/images/methodology-banner-sm.webp" />
+          <source type="image/jpeg" srcset="/static/images/methodology-banner-sm.png" />
+          <img src="/static/images/methodology-banner.png" class="content-banner" alt="" width="1200" height="984" loading="eager" />
+        </picture>
+        <h2 id="overview"><a href="#overview" class="anchor-link">Краткий обзор</a></h2>
+
+        <p>
+          The Web Almanac — это проект, организованный <a href="https://httparchive.org">HTTP Archive</a>. HTTP Archive был запущен в 2010 Стивом Садерсом (Steve Souders), чтобы отслеживать развитие веба. Каждый месяц этот сервис анализирует содержимое миллионов веб-страниц и записывает терабайты метаданных в <a href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data">BigQuery</a> для последующего анализ.
+        </p>
+
+        <p>
+          Задача Web Almanac состоит в том, чтобы стать ежегодным хранилищем общедоступных сведений о состоянии веба. Наша цель — понятно рассказать о данных, собранных HTTP Archive, для этого мы приглашаем профильных экспертов.
+        </p>
+
+        <p>
+          Выпуск Web Almanac в 2020 году поделён на четыре части: контент, пользовательские ощущения, публикация и распространение. В каждой части есть несколько глав, которые рассматривают главную тему с разных точек зрения. Например, во второй части пользовательские ощущения в соответствующих главах рассматриваются с разных сторон, а именно: производительности, безопасности и доступности.
+        </p>
+    </section>
+
+    <section>
+        <h2 id="dataset"><a href="#dataset" class="anchor-link">О наборе данных</a></h2>
+
+        <p>
+          Набор данных HTTP Archive постоянно растёт каждый месяц. Для издания Web Almanac 2020 года, если в главе не указано иное, все метрики были взяты из сканирования сайтов за август 2020 года. Эти показатели можно <a href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">свободно запросить</a> из таблиц с префиксом <code>2020_08_01</code> на BigQuery.
+        </p>
+
+        <p>
+          Все метрики, представленные в Web Almanac, можно получить из набора данных на BigQuery. Вы можете посмотреть запросы, используемые во всех главах, в нашем <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">репозитории на GitHub</a>.
+        </p>
+
+        <p class="note">
+          Обратите внимание, что некоторые запросы могут быть достаточно большими и <a href="https://cloud.google.com/bigquery/pricing">дорогостоящими</a> для выполнения, так как оплата за пользование BigQuery начинается после обработки одного терабайта данных. Для того, чтобы помочь вам контролировать расходы, прочитайте пост Тима Кадлека (Tim Kadlec) <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.
```

Comment:
Same comment above about adding `lang="en"` and question as to whether the title should be translated or not?

```suggestion
          Обратите внимание, что некоторые запросы могут быть достаточно большими и <a href="https://cloud.google.com/bigquery/pricing">дорогостоящими</a> для выполнения, так как оплата за пользование BigQuery начинается после обработки одного терабайта данных. Для того, чтобы помочь вам контролировать расходы, прочитайте пост Тима Кадлека (Tim Kadlec) <a lang="en" href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.
```